test/suite/test_cc01.py

```diff
@@ -77,6 +77,14 @@ def check(self, check_value, uri, nrows, read_ts):
         session.rollback_transaction()
         self.assertEqual(count, nrows)
 
+    def populate(self, uri, start_key, num_keys, value):
```

Comment:
Why you need to add this?

Reply:
It's part of the base class, we can now use `self.populate()` in any `cc` test which I am doing in this PR.